test/e2e/tests/multichain/asset-list.spec.ts

```diff
@@ -14,13 +14,15 @@ import AssetListPage from '../../page-objects/pages/asset-list';
 const NETWORK_NAME_MAINNET = 'Ethereum Mainnet';
 const LINEA_NAME_MAINNET = 'Linea Mainnet';
 const LOCALHOST = 'Localhost 8545';
+const POLYGON_NAME_MAINNET = 'Polygon';
 const BALANCE_AMOUNT = '24.9956';
 
-function buildFixtures(title: string) {
+function buildFixtures(title: string, chainId: number = 1337) {
   return {
```

Comment:
I'm still not sure how this used to work before without issues 🤔

Reply:
It appears that the feature flag was not enabled in the CI environment, which is the primary reason we didn't encounter this issue earlier.